AI/BattleAI/BattleAI.cpp

```diff
@@ -53,7 +53,7 @@ BattleAction CBattleAI::activeStack( const CStack * stack )
 		if(stack->hasBonusOfType(Bonus::SIEGE_WEAPON) && stack->hasBonusOfType(Bonus::HEALER))
 		{
 			auto healingTargets = cb->battleGetStacks(CBattleInfoEssentials::ONLY_MINE);
-			std::map<int, const CStack*> woundHpToStack;
+			std::map<int, const CStack *> woundHpToStack;

```

Comment:
This is totally intended and I don't see anything wrong with that since ```*>``` on it's own have no special meaning in C++ as far as I aware. Space after the ptr star is questionable, but we'll have space before it for sure.